src/PKSim.Infrastructure/PKSimConfiguration.cs

```diff
@@ -33,6 +36,16 @@ public PKSimConfiguration()
          TemplateSystemDatabasePath = LocalOrAllUsersPathForFile(CoreConstants.TEMPLATE_SYSTEM_DATABASE);
          TemplateUserDatabaseTemplatePath = LocalOrAllUsersPathForFile(CoreConstants.TEMPLATE_USER_DATABASE_TEMPLATE);
          DefaultTemplateUserDatabasePath = CurrentUserFile(CoreConstants.TEMPLATE_USER_DATABASE);
+         
+         RemoteTemplateFolderPath = Path.Combine(CurrentUserFolderPath, CoreConstants.REMOTE_FOLDER_PATH);
+         if (!DirectoryHelper.DirectoryExists(RemoteTemplateFolderPath))
+            DirectoryHelper.CreateDirectory(RemoteTemplateFolderPath);
+
+         //The summary file is installed in All user paths but needs to be copied in current user to be updated 
+         var installTemplatePath = LocalOrAllUsersPathForFile(CoreConstants.REMOTE_TEMPLATE_SUMMARY);
+         RemoteTemplateSummaryPath = CurrentUserFile(CoreConstants.REMOTE_TEMPLATE_SUMMARY);
+         if (!FileHelper.FileExists(RemoteTemplateSummaryPath))
+            FileHelper.Copy(installTemplatePath, RemoteTemplateSummaryPath);
```

Comment:
Copying the file from install folder to user folder. When we implement refresh on demand, this is the file that will be updated